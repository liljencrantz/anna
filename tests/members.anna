
type MyType {
    def meth()
    {
	print("Foo.poo()\n");	
    }
    var Int mem;
}

def main(){
    g := MyType();
    g.meth();
    print(g.mem, "\n");
    g.mem=5;
    print(g.mem, "\n");
}

