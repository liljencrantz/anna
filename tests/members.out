Foo.poo()
null
5
